pkg/aws/helpers.go

```diff
@@ -27,7 +27,24 @@ import (
 	rprtr "github.com/openshift/rosa/pkg/reporter"
 )
 
-var RoleNameRE = regexp.MustCompile(`^[\w+=,.@-]+$`)
+const RoleNameREString = `^[\w+=,.@-]+$`
+
+var RoleNameRE = regexp.MustCompile(RoleNameREString)
+
+func getREOfAllAccRolesSuffixes() string {
+	suffixes := make([]string, 4)
+	for _, role := range AccountRoles {
+		suffixes = append(suffixes, role.Name)
+	}
+	return strings.Join(suffixes, "|")
+}
```

Comment:
Should have been 0, 4... I end up making this mistake from time to time in go still ...

I want it to be through the AccountRoles map so that if we ever change it at some point there is no need to refactor anything or end up missing something that needed to be changed